allofplos/update.py

```diff
@@ -0,0 +1,4 @@
+from .corpus.plos_corpus import main
```

Comment:
If we're going to do this, I'd prefer that the meat of this functionality actually be moved into this file and rely on the `corpus` module to provide the actual APIs it uses internally. This kind of redirection is getting to be a bit gratuitous. But if this would include a true refactor (making `python -m allofplos.corpus.plos_corpus` invalid, then I'd be for this.

Reply:
I'll note that I've been trying to push us to have a deeper package structure, and so introducing another top level module isn't exactly a great idea in my opinion. 

However, I think it's important to separate the applications from the APIs, and this is effectively an application style command, so that's why I'm in favour of adding this at the cost of a flatter package structure to gain the isolation of purpose.